packages/grafana-ui/src/components/ValuePicker/ValuePicker.tsx

```diff
@@ -52,6 +55,7 @@ export function ValuePicker<T>({
           icon={icon || 'plus'}
           onClick={() => setIsPicking(true)}
           variant={variant}
+          fill={fill}
           fullWidth={isFullWidth}
           aria-label={selectors.components.ValuePicker.button(label)}
```

Comment:
we should extend `ValuePicker`s props to accept `aria-label` as well and pass an `aria-label` in `OrganizationPicker`, something like `Change organization`

we can then make this something like `aria-label={selectors.components.ValuePicker.button(aria-label ?? label)}`